larray/io/excel.py

```diff
@@ -455,9 +431,77 @@ def load(self, header=True, convert_float=True, nb_index=None, index_col=None):
                 return LArray(list_data)
 
     # XXX: remove this function?
-    def open_excel(filepath=None, **kwargs):
-        return Workbook(filepath, **kwargs)
+    def open_excel(filepath=None, mode='a', visible=None, silent=None, app=None):
+        return Workbook(filepath, mode, visible, silent, app)
 else:
-    def open_excel(filepath=None, **kwargs):
+    def open_excel(filepath=None, mode='a', visible=None, silent=None, app=None):
         raise Exception("open_excel() is not available because xlwings "
                         "is not installed")
+
+open_excel.__doc__ = \
+"""
+Open an Excel workbook
+
+Parameters
+----------
+filepath : None, int or str, optional
+    path to the Excel file. The file must exist if mode is set to 'r' or 'w'. 
+    Use None for a new blank workbook, -1 for the last active
+    workbook. Defaults to None.
+mode : {'r', 'w', 'a'}, optional
+    the mode to open the file. It can be one of the following:
+
+    * 'r': Read; similar to 'a', but the file must already exist 
+      (the read-only mode is currently not supported by xlwings).
+    * 'w': Write; a new file is created (an existing file with the same name would be deleted).
+    * 'a': Append; an existing file is opened for reading and writing, and if the file does not exist it is created.
+    * 'r+': currently idem as 'r'.
```

Comment:
I guess it will. No idea when though. At worse (if we actually need the functionality before it is implemented), we could implement it ourselves and submit a PR to them.